src/parsing/index.ts

```diff
@@ -115,12 +115,12 @@ export interface ColumnAliasSuggestion {
     types: string[];
 }
 
-export function parseGenericSql(queryBeforeCursor: string, queryAfterCursor: string): ParseResult {
+export function parseGenericSql(queryBeforeCursor: string, queryAfterCursor: string = ''): ParseResult {
     let parser = genericAutocompleteParser as Parser;
     return parser.parseSql(queryBeforeCursor, queryAfterCursor);
 }
 
-export function parsePostgreSql(queryBeforeCursor: string, queryAfterCursor: string): ParseResult {
+export function parsePostgreSql(queryBeforeCursor: string, queryAfterCursor: string = ''): ParseResult {
```

Comment:
It's solves problem with parse query without queryAfterCursor argument, and we don't need separate methods

Reply:
Oh, i got it